sdk/communication/azure-communication-phonenumbers/pom.xml

```diff
@@ -0,0 +1,174 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-communication-phonenumbers</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.0-beta.4</version> <!-- {x-version-update;com.azure:azure-communication-phonenumbers;current} -->
+
+  <name>Microsoft Azure client phone numbers library for communication</name>
+  <description>
+    This package contains APIs for managing phone numbers in Microsoft Azure Communication Services.
+    For this release, see notes - https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-phonenumbers/README.md and https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-phonenumbers/CHANGELOG.md.
+    Microsoft Azure Communication Administration quickstart - https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-java
+  </description><!-- TODO: Update documents for Phone Numbers -->
+  <licenses>
+    <license>
+        <name>The MIT License (MIT)</name>
+        <url>http://opensource.org/licenses/MIT</url>
+        <distribution>repo</distribution>
+    </license>
+</licenses>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <src.dir>src/main</src.dir>
+    <test.dir>src/test</test.dir>
+    <jacoco.min.linecoverage>0.00</jacoco.min.linecoverage>
```

Comment:
you will add some coverage number in future as development progresses .

Reply:
Fixed thank you